warehouse/profiles.yml

```diff
@@ -1,11 +1,11 @@
 calitp_warehouse:
-  target: staging
+  target: prod
   outputs:
     prod:
       &prod
       execution_project: cal-itp-data-infra
       database: cal-itp-data-infra
-      schema: staging
+      schema: views
```

Comment:
I wouldn't change this default, I think? Helps to avoid accidentally exposing models if we change around the schemas in dbt_project.yml.